datahub-web-react/src/app/entity/shared/EntityDropdown/EntityDropdown.tsx

```diff
@@ -197,10 +200,21 @@ function EntityDropdown(props: Props) {
                                 </MenuItem>
                             </StyledMenuItem>
                         )}
+                        {menuItems.has(EntityMenuItems.CLONE) && (
```

Comment:
how do you feel about moving this item down below the move and delete buttons? clone feels like an additional "nice to have" while move and delete feel more like required functionality.